### DISCUSSION TITLE
Add CREATE-PLAY.md as a page in reactplay.io

### DISCUSSION THREAD

**Discussion by MOHAMED-EHAB-DEV** [reactplay/react-play#268] [Ideas]
Hello Everyone

I have an idea that I would like to share with you

Add CREATE-PLAY.md as a page in reactplay.io

Something like reactplay.io/create-play

Do you have any other idea ?

**Comment by atapas** [comment]
We are bringing the entire Play-Create experience to UI. So these steps are temporary. We may need a documentation page to get our Coding Checklist. PR check list in that.
